### PR TITLE
chore: update cluster-autoscalers to latest patches

### PR DESCRIPTION
**Pull request by mboersma**
**Reason for Change**:

See https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.18.3
and https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.17.4
and https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.16.7
and #3996

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [x] No
- [ ] Yes

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by mboersma** [comment]
/hold

This has a temporary testing commit that needs to be discarded before merging.

**Comment by mboersma** [comment]
New autoscalers passed our CI for 1.16 to 1.18:

image

I'll strip off the testing commit and then this should be ready for review.

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Pull request merged**
